channels/rail/client/rail_orders.c

```diff
@@ -1597,7 +1597,11 @@ UINT rail_send_client_window_move_order(railPlugin* rail, const RAIL_WINDOW_MOVE
 	error = rail_write_client_window_move_order(s, windowMove);
 
 	if (error == ERROR_SUCCESS)
+	{
+		WLog_Print(rail->log, WLOG_TRACE, "sending new RAIL window corners: 0x%x (%d, %d), (%d, %d)",
```

Comment:
Please check the format string and use the appropriate `PRIx32` or similar C macros

Reply:
Ok (I wasn't familiar with those macros).